samples/cpp/benchmark_app/main.cpp

```diff
@@ -398,6 +396,10 @@ int main(int argc, char* argv[]) {
                 // for CPU execution, more throughput-oriented execution via streams
                 setThroughputStreams();
                 set_infer_precision();
+
+                if (isFlagSetInCommandLine("dopt")) {
+                    device_config.emplace(ov::intel_cpu::denormals_optimization(true));
```

Comment:
@nkogteva do we need this option in BA? Or we can suggest using a generic config file where all such properties can be put

Reply:
@ilya-lavrenov It can be easily specified using config file, I suggest don't add this parameter as sepatate command line option.

Reply:
Thanks for the advice. From my perspective, this BA option is convenient to use with message explanation with -help. Users can easily enable/disable it when they make test with BA to decide whether use this optimization, no additional document needed and don't care about how to write config file and where to place. And sometimes there are perf changes due to this opt, user maybe need sometime to root cause it in config file. When they decide to enable it, they can use config file to specify in their application. correct me please if I misunderstanding sth.

Reply:
My recommendation is to remove explicit command line option. We are trying to keep BA options list simple and generic for all devices. 
From my perspective our goal is to enabled denormals optimization by default on CPU in the next release. Given that this BA option will lose all meaning. I hope we will be able to live with json config  until next release.

Reply:
Thanks, I see. removed BA option.